PalasoUIWindowsForms/ImageToolbox/ArtOfReadingChooser.cs

```diff
@@ -218,10 +227,98 @@ private void ArtOfReadingChooser_Load(object sender, EventArgs e)
 		   // _searchTermsBox.Text = @"flower";
 			_searchButton_Click(this,null);
 #endif
-
+			var searchControl = _searchTermsBox.TextBox.TheControl;
+			if (searchControl is TextBox)
+				searchControl.MinimumSize = _searchTermsBox.Size;
 			_messageLabel.BackColor = Color.White;
 		}
 
+		private void SetMessageLabelText()
+		{
+			var msg = "In the box above, type what you are searching for, then press ENTER.".Localize("ImageToolbox.EnterSearchTerms");
+			var searchLang = "English + Indonesian";
+			if (_searchLanguageMenu.Visible)
+				searchLang = _searchLanguageMenu.ToolTipText;
```

Comment:
Why is the search language stored in the menu's tooltip?? and when and why would the menu not be visible??